### ISSUE TITLE
Adding MobxAngularModule

### ISSUE THREAD

**Issue by rhjennings** [PacktPublishing/Hands-On-App-Development-with-Ionic#7]
After I add the MobxAngularModule, I'm getting the following error:

[ng] ERROR in node_modules/mobx/lib/types/observableset.d.ts(2,72): error TS1005: ';' expected.
[ng] node_modules/mobx/lib/types/observableset.d.ts(2,74): error TS1011: An element access expression should take an argument.

If I checkout the section02 code it works, but adding it manually does not. So, I'm thinking there is something missing.

My mobx version: ^5.15.4
mobx-angular: ^4.1.0

Section02 mobx version: ^5.10.1
mobx-angular: ^3.0.1

I changed those versions to match, but still got the same error.

**Comment by AhsanAyaz** [comment]
Hey @rhjennings, thanks for reporting the issue.
I was also able to reproduce the issue you mentioned.
![image](https://user-images.githubusercontent.com/9844254/83365016-99979180-a3a5-11ea-8300-16cc9231bce0.png)

The reason is with the instructions in the slides, we're installing the latest version of the packages, which now have been quite updated. The command to run should rather be as follows for the exact versions:
`npm install --save mobx@5.10.1 mobx-angular@3.0.1`

I've asked @PacktNishanthS to have this reflected in the video as well. Let me know if that solves your issue.

Thanks again 🙂

**Comment by rhjennings** [comment]
Yes, after running the npm install with those versions, the app will now run.
